README.md

```diff
@@ -44,12 +44,14 @@ Select:
 
 ```sh
 export CAPM3_VERSION=v1alpha4
+export CAPI_VERSION=v1alpha3
 ```
 
 or
 
 ```sh
 export CAPM3_VERSION=v1alpha5
+export CAPI_VERSION=v1alpha4
 ```
 
 Select:
```

Comment:
We do not need lines starting from here till L69 since you are already suggesting to select CAPI versions right before these lines

Reply:
True will be removed